src/const.js

```diff
@@ -14,5 +14,24 @@ const SortType = {
   OFFERS: 'Offers'
 };
 
+const UserAction = {
+  UPDATE_POINT: 'UPDATE_POINT',
+  ADD_POINT: 'ADD_POINT',
+  DELETE_POINT: 'DELETE_POINT',
+};
+
+const UpdateType = {
+  PATCH: 'PATCH',
+  MINOR: 'MINOR',
+  MAJOR: 'MAJOR',
+};
+
+const EmptyListPhrase = {
+  'everything':'Click New Event to create your first point',
+  'past':'There are no past events now',
+  'present':'There are no present events now',
+  'future':'There are no future events now'
```

Comment:
Отличное решение 👍, ещё можно вот так добавить:
```suggestion
  [FilterType.EVERYTHING]: 'Click New Event to create your first point',
  [FilterType.PAST]: 'There are no past events now',
  [FilterType.PRESENT]: 'There are no present events now',
  [FilterType.FUTURE]: 'There are no future events now'
```